app/assets/stylesheets/forever_style_guide/globals/_breakpoints.scss

```diff
@@ -45,7 +45,7 @@ $screen-xl-max:              ($screen-xxl-min - 1) !global;
 
 
 //** Point at which the navbar becomes uncollapsed.
-$grid-float-breakpoint: 920px!global;
+$grid-float-breakpoint: $screen-lg !global;
```

Comment:
two (competing) priorities here would be to see if we can't get the breakpoint down further to increase the likelihood of optimal display and also to limit the likelihood the ambassador name is ellipsized. See the other points regarding primary type size, padding, etc. Just noting here since this would be affected.